drivers/accel/adxl355/adxl355.c

```diff
@@ -612,10 +630,22 @@ int adxl355_get_temp(struct adxl355_dev *dev, struct adxl355_frac_repr *temp)
 	int ret;
 
 	ret = adxl355_get_raw_temp(dev, &raw_temp);
-	if (!ret)
-		temp->integer = no_os_div_s64_rem(adxl355_temp_conv(raw_temp),
-						  ADXL355_TEMP_OFFSET_DIV*ADXL355_TEMP_SCALE_FACTOR_DIV,
-						  &(temp->fractional));
+	if (!ret) {
```

Comment:
If we can simplify code by reducing the amount of nested statements we should do so. Here it's better to:
```
if (ret)
    return ret;
// continue here
```
instead of:
```
if (!ret) {
    // continue here
}
```